standards/javascript.md

```diff
@@ -141,41 +141,40 @@ same page.
 ##### Bad
 
 ```js
-var foo = 1,
+const foo = 1,
     bar = 2;
```

Comment:
I know this is the bad example, but it's demonstrating the inconsistency. Indent `bar` here to match `foo`; we're not animals.

Reply:
See the content between the two code blocks.

Reply:
Sorry, I wrote that poorly. Let me try again:

`bar` was aligned with `foo` before this change. It should still be.

Reply:
Ohhh, gotcha, I'll fix that in the next branch